app/tests/integration/Forms/Form.test.tsx

```diff
@@ -13,6 +13,25 @@ import {FormJson} from 'next-env';
 import {generateFakeSchemaData} from 'tests/integration/json-schema-utils';
 import MockRouter from 'tests/integration/MockRouter';
 
+// Product & Fuel were not rendering anything (because we mess with the schema in ProductRowIdField & those values are not making it in)
+// This mock allows the data to show up (with the name of the product or fuel as 'one')
+jest.mock('components/SearchDropdown', () => {
+  return {
+    __esModule: true,
+    default: () => {
+      return 'one';
+    }
+  };
+});
+
+// Todo: Mock so that the entire flow works properly
+
+// jest.mock('components/Forms/SearchDropdownWidget', () => ({onChange,
+//   schema,
+//   id,
+//   placeholder,
+//   value}) => {return <mock-SearchDropdownWidget schema={{...schema, enums: [1,2], enumNames: ['one', 'two'], state: ['active', 'active']}} />}
+// );
```

Comment:
Does this belong in this PR? Please make a separate PR to fix the tests if it doesn't

Reply:
It fixes the production & fuel tests returning nothing in the snapshot, so kind of? You can see the effect in the snapshot diff, but the issue was not indtroduced in this PR.